docs/gcc-Tips.rst

```diff
@@ -32,27 +32,34 @@ Some explanatory pictures:
 .. image:: ./_static/images/gcc-table.png
    :align: center
    :target: ./_static/images/gcc-table.png
+   :alt: picture comparing depth with number of threads with each implementation where red color means bad combination and greener areas mean good combination
```

Comment:
I think these unrelated changes may have shown up because you created https://github.com/microsoft/LightGBM/pull/4175 from `master` instead of a separate branch on your fork.

Could you please remove them?

I think you'll need to do the following.

1. close https://github.com/microsoft/LightGBM/pull/4175
2. create a new branch from your local `master`

```shell
git checkout master
git checkout -b docs/gcc-alt-txt
git push origin docs/gcc-alt-txt
```

3. Open a new pull request for the `gcc-tips` documentation, from the branch `docs/gcc-alt-txt`
4. reset `master` locally and on your fork to LightGBM's `master`

```shell
# delete local master branch
git branch -D master

# add LightGBM official repo as another remo
git remote add upstream git@github.com:microsoft/LightGBM.git

# get master branch from LightGBM official repo
git fetch upstream master
git checkout master

# overwrite your fork's "master" branch with the official LightGBM one
git push origin master --force
```

5. Rebase this branch to that new `master`

```shell
git checkout akshita_fstrings
git rebase master
git push origin akshita_fstrings --force
```

Let me know if you have any questions about that, we're here to help!

Reply:
Doing that right away!